### PR TITLE
Navigation Skip Links

### PR DESCRIPTION
**Pull request by curtisadams28**
[ADO Card](https://dev.azure.com/S72/SHIFT72/_workitems/edit/4487)

The skip link navigation appears when tabbed (focused) on the links.

image

### CODE REVIEW

File: site/templates/application/application.jet [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {{import "./head/font.jet" }}
 {{import "./head/seo.jet" }}
 {{import "./nav/nav.jet"}}
+{{import "./nav/skip-links.jet"}}
```
Comment (PeculiarGoat) [inline site/templates/application/application.jet:4]: I'd argue that this shouldn't go in `/nav` because it's not part of _the_ nav.

**Review by PeculiarGoat — APPROVED** [review]
Looks good.  I'm just unsure about using the word `nav` in your classes since this technically isn't part of the nav.

**Comment by PeculiarGoat** [comment]
Don't forget the changelog!

**Comment by curtisadams28** [comment]
Closing as this requires an elab.

**Pull request closed without merging**